hybrasyl/Game.cs

```diff
@@ -344,7 +343,9 @@ public static void Main(string[] args)
                         stipulationWriter.Write($"Welcome to Hybrasyl!\n\nThis is Hybrasyl (version {Assemblyinfo.Version}).\n\nFor more information please visit http://www.hybrasyl.com");
                         if (string.IsNullOrEmpty(Motd))
                         {
-                            Motd = ControlService.GetMotd();
+                            //TODO: Rework for .net core
+                            //Motd = ControlService.GetMotd();
+                            Motd = "Generic MOTD.";

```

Comment:
TBH this is fine / not a big deal since you can still set via server.msg

Reply:
Correct.